modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.es.js

```diff
@@ -82,9 +82,11 @@ export {default as createPortletURL} from './liferay/util/portlet_url/create_por
 export {default as createRenderURL} from './liferay/util/portlet_url/create_render_url.es';
 export {default as createResourceURL} from './liferay/util/portlet_url/create_resource_url.es';
 
-// loadModule API
+// Client Extensions API
 
-export {getModuleAndSymbolNames, loadModule} from './module';
+export {getModuleAndSymbolNames} from './utils/client_extensions/getModuleAndSymbolNames';
+export {default as loadClientExtensions} from './utils/client_extensions/loadClientExtensions';
+export {loadModule} from './utils/client_extensions/loadModule';
```

Comment:
- I moved `loadClientExtensions` to `frontend-js-web`, as it is now used from two modules.
- I put all three related utils under `utils`, matching (still informal) good structure practices
- I put all three related utils under the `client_extensions` bucket. Technically `loadModule` can be used outside of CETs, but I don't think we want to promote it.

Reply:
What about putting `loadModule` and `getModuleAndSymbolNames` inside `loadClientExtensions.ts` as private functions? Would that break anything?

Because if we internalize them people won't be able to access them and won't be able to couple their stuff with `frontend-js-web` only because they found a very handy function there.

Technically this is a deeper iteration on top of your:

>  I put all three related utils under the client_extensions bucket. Technically loadModule can be used outside of CETs, but I don't think we want to promote it.

Reply:
We can make `getModuleAndSymbolNames` a private function :+1: 

`loadModule` is unfortunately used directly in FDS, [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/Filter.tsx#L66) and [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js#L425). I believe we can refactor it out, replacing it with `loadClientExtensions`, but that's outside of scope of this task.